nbtlib/path.py

```diff
@@ -0,0 +1,177 @@
+from .tag import *
+from .literal.parser import Parser,tokenize
+__all__ = ('NBT_Path',)
+
+import re
+def slice_nbt(literal):
+    parser = Parser(tokenize(literal))
+    tag = parser.parse()
+
+    cursor = parser.token_span[1]
+    leftover = literal[cursor:]
+
+    return tag,leftover
+
+
+def match(obj,pattern):
+	for i in Int.all_tags.values():
```

Comment:
Style issue: I don't recommend to use both tabs and spaces for indent in one file. It seems that the owner prefers 4 spaces.

Reply:
done.